posts/2022-06-24-rust-unconference.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Announcing The RustConf PostConf UnConf"
+author: Jane Lusby, on behalf of The Rust Project Teams
+---
+
+Hello Rust community!
+
+We're excited to announce that the Rust project teams be hosting an
+unconference[^1] the day after RustConf.
+
+![RustConf PostConf UnConf promo image](../../../images/2022-06-24-rust-unconference/Eventbrite_Unconf.jpg)
+
+The PostConf Unconf will be dedicated to the Rust project and will be a
+fantastic opportunity for users, contributors, and maintainers to network and
+discuss the project's development.
+
+There will be no set agenda; instead, attendees will decide what will be
+discussed together and can move freely between sessions to find ones in which
+they can contribute most effectively based on their individual interests and
+needs.
+
+To get the most out of the unconference, jot down your thoughts ahead of time
+and bring them ready to share. We will also setup a channel in the RustConf
+Discord for folks to communicate and make preliminary, informal plans.
+
+If you plan to attend, please register as soon as possible to help us plan
+appropriately. If space is limited, project participants and conference
+attendees will be given preference. Registration is free and open to everyone,
+but we require either a RustConf registration ID or a $10 deposit at signup to
```

Comment:
Is the deposit given back if the person does show up?

Reply:
Yes